### PR TITLE
Android structure project

### PR DESCRIPTION
**Pull request by danielpassos**
I have moved the project to a new Android structure. It will allow us add submodules like wear.

PS: I want to play with wear in a near future

### CODE REVIEW

**Comment by danielpassos** [comment]
@anuj1708 Can you check if everything still working and I did not missed any file?

**Comment by openwarrior** [comment]
@danielpassos Please do `mvn license:format`

**Comment by danielpassos** [comment]
Just found and problem, and also need to fix the travis problem with wear project. I'll close it and send a new one

**Comment by danielpassos** [comment]
Build fixed!

File: gradlew [hunk 1]
```diff
@@ -61,9 +56,9 @@ while [ -h "$PRG" ] ; do
     fi
 done
 SAVED="`pwd`"
-cd "`dirname \"$PRG\"`/" >&-
+cd "`dirname \"$PRG\"`/" >/dev/null
```
Comment (openwarrior) [inline gradlew:59]: I am not getting what are these for.

**Comment by openwarrior** [comment]
Looking everything good to me. Nothing is broken

File: gradlew (hunk 1 above)
Comment (danielpassos) [inline gradlew:59]: Me either, anyway, it's from Gradle, I'll take a look and tell you

**Pull request merged**